docs/rules/space-in-parens.md

```diff
@@ -18,80 +20,76 @@ This rule will enforce consistency of spacing directly inside of parentheses, by
 
 ## Options
 
-There are two options for this rule:
+There are two basic options for this rule:
 
 * `"always"` enforces a space inside of parentheses
 * `"never"` enforces zero spaces inside of parentheses (default)
 
```

Comment:
I moved a large block of text from Examples > Exceptions in to the Options section, to ensure that all options are described in one section rather than scattered through file.